tests/src/test/scala/common/Wsk.scala

```diff
@@ -142,17 +142,20 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
      *
      * @param namespace (optional) if specified must be  fully qualified namespace
      * @param expectedExitCode (optional) the expected exit code for the command
+     * @param verboseFlag (optional) if true prints full verbose output
      * if the code is anything but DONTCARE_EXIT, assert the code is as expected
      */
     def list(
         namespace: Option[String] = None,
         limit: Option[Int] = None,
         nameSort: Option[Boolean] = None,
-        expectedExitCode: Int = SUCCESS_EXIT)(
+        expectedExitCode: Int = SUCCESS_EXIT,
+        verboseFlag: Option[Boolean] = None)(
             implicit wp: WskProps): RunResult = {
         val params = Seq(noun, "list", resolve(namespace), "--auth", wp.authKey) ++
             { limit map { l => Seq("--limit", l.toString) } getOrElse Seq() } ++
-            { nameSort map { n => Seq("--name-sort") } getOrElse Seq() }
+            { nameSort map { n => Seq("--name-sort") } getOrElse Seq() } ++ 
+            { verboseFlag map { v => Seq("-v")} getOrElse Seq()}
```

Comment:
use the cli() function provided `verbose` argument.

Reply:
https://github.com/apache/incubator-openwhisk/blob/master/tests/src/test/scala/common/Wsk.scala#L1041